lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -172,6 +172,32 @@
     default: []
     choices: ['Launch', 'Terminate', 'HealthCheck', 'ReplaceUnhealthy', 'AZRebalance', 'AlarmNotification', 'ScheduledActions', 'AddToLoadBalancer']
     version_added: "2.3"
+  metrics_collection:
+    description:
+      - Enable group metrics collection for the auto scaling group.
+    default: False
+    required: False
+    version_added: "2.4"
+  metrics_granularity:
+    description:
+      - The granularity to associate with the metrics to collect. The only valid value is 1Minute.
```

Comment:
Please remove `The only valid value is 1Minute.`

That is covered by the `choices:` line